retrofit/src/main/java/retrofit/DefaultConverter.java

```diff
@@ -0,0 +1,18 @@
+package retrofit;
+
+import com.squareup.okhttp.RequestBody;
+import com.squareup.okhttp.ResponseBody;
+import java.io.IOException;
+import java.lang.reflect.Type;
+
+final class DefaultConverter implements Converter {
```

Comment:
Are you saying a converter is then required? As it stands now it'd be perfectly fine to use Retrofit with `RequestBody` and `ResponseBody` as your only non-primitive, non-String types.